### DISCUSSION TITLE
Dart analysis error

### DISCUSSION THREAD

**Discussion by defro** [kevlatus/flutter_fortune_wheel#70] [General]
Hello,

I've got a dart analysis error on this line : https://github.com/kevlatus/flutter_fortune_wheel/blob/acf17fcbed0c03da625f40d0e1ae90a6d7ab1bd0/example/lib/main.dart#L89

This is the error:
error: The method 'AppRoute' isn't defined for the type '_DemoAppState'. (undefined_method at [flutter_fortune_wheel_example] lib/main.dart:88)

Can you help me ?

Regards,
Joël

**Comment by kevlatus** [comment]
Hi @defro, when you want to build the code locally, you need to run the code generator. This is achieved by exectuting `flutter pub run build_runner build` in the _example_ directory.
